doc/usage.md

```diff
@@ -199,6 +199,37 @@ CodeCompass_webserver -w <workdir> -p <port> -d <connection_string>
 
 For full documentation see `CodeCompass_webserver -h`.
 
+### Enabling HTTPS (SSL/TLS) secure server
+
+By default, CodeCompass start a conventional, plain-text HTTP server on the
```

Comment:
```suggestion
By default, CodeCompass starts a conventional, plain-text HTTP server on the
```

Reply:
Nice catch, thanks. I need Grammarly in my editor... :frowning: I found a few more weird language sentences too! *"workspace of the directory"* :confounded: